spec/lib/chat_review_queue_spec.rb

```diff
@@ -27,6 +27,56 @@
       }.to raise_error(Discourse::InvalidAccess)
     end
 
+    it "raises an error if the user already flagged the post" do
```

Comment:
I know there are a lot of things moving at the moment regarding how we write our specs but usually when there’s an `if` in a description it’s a good indicator that a context should be used. This allows for easier setup and grouping of things like related `let`. Not sure where we’re at on this front in the chat project though :slightly_smiling_face: 
Anyway, we could then rewrite this block as:
```ruby
context "when the user already flagged the post" do
  let(:second_flag_result) { queue.flag_message(message, guardian, ReviewableScore.types[:off_topic]) }

  before do
    queue.flag_message(message, guardian, ReviewableScore.types[:spam])
  end

  it "returns an error" do
    expect(second_flag_result).to include success: false, errors: I18n.t("reviewables.already_handled")
  end
end
```
I also named the subject (`queue`) as it’s recommended to do so in the rspec style guide.

Reply:
Looks much cleaner. Thanks for the suggestion!